tests/date.test.js

```diff
@@ -0,0 +1,84 @@
+import { formatDate, toDate } from '../utils/date'
+import dayjs from 'dayjs'
+
+describe('date', () => {
+    const currentDate = dayjs().format('DD.MM.YYYY')
+    let dateFormat
+    const date = {
+        day: 25,
+        firstDay: '0' + 1,
+        month: 12,
+        time: {
+            opt1: '13:15:40.000000',
+            opt2: 'T12:15:40.000Z'
+        },
+        year: 2019
+    }
+    const dates = {
+        date: new Date(`${date.year}-${date.month}-${date.day}${date.time.opt2}`),
+        number: 1577276140000,
+        string: `${date.year}-${date.month}-${date.day} ${date.time.opt1}`
```

Comment:
```suggestion
    const dateTypes = {
        exactDate: new Date(`${date.year}-${date.month}-${date.day}${date.time.opt2}`),
        timestampDate: 1577276140000,
        IsoDate: `${date.year}-${date.month}-${date.day} ${date.time.opt1}`
```
I would like suggest to take another naming here to understand what type of dates here are mentioned or why you mentioned this type of dates